api/config.go

```diff
@@ -57,7 +57,7 @@ type Config struct {
 	SwapEnabled             bool           // whether SWAP incentives are enabled
 	SwapPaymentThreshold    uint64         // honey amount at which a payment is triggered
 	SwapDisconnectThreshold uint64         // honey amount at which a peer disconnects
-	SwapInitialDeposit      uint64         // initial deposit amount to the chequebook
+	SwapDeposit             uint64         // deposit amount to the chequebook
```

Comment:
Depositing is not any more atomic with deploying. I adjusted the boot-sequence slightly to ask *always* for a deposit amount, even if you are not deploying. Hence, `initial` is not anymore fully correct

Reply:
*Always* asking seems worse. I would suggest to leave the old boot-sequence. And just do deployment and deposit back-to-back during startup. We'll have to implement proper refill logic later but without tying it to node startup.

Reply:
I have been thinking about this, but I don't think the original boot-sequence is still possible. The major reason is that deploying and depositing is not any more atomic: previously those happened in one transaction, but right now they are two separate transactions, making it possible that deploying works, but depositing fails. I think it would not be nice for a user to want to do an initial deposit, see that it does not work and then have to figure out how to do the deposit via a different way.

The other reason is UX-related. I believe that the moment you boot up a Swarm node is the natural time to do a refill if you think that the `AvailableBalance` is not sufficient for the time you intend to use your node. 

This option can be silenced by adding the flag `--swap-deposit 0`.

That being said, I will make one more commit for the doing the `--swap-deposit 0` option properly and the changing the name `swap-initial-deposit` everywhere.

Reply:
I added an additional flag `--swap-no-deposit` to be able to silence the prompt. Regardless of the outcome of this discussion, I would say this flag is needed anyways.

We can consider removing the prompt at all when a correct warning message is in place when availableBalance is close to zero (https://github.com/ethersphere/swarm/issues/1885)